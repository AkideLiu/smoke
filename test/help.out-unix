Smoke: a framework for testing most things from the very edges.

Usage: smoke-exe [--command ARG] ([-c|--color] | [--no-color]) TEST-LOCATION...

Available options:
  --command ARG            Specify or override the command to run
  -c,--color               Color output
  --no-color               Do not color output
  -h,--help                Show this help text
