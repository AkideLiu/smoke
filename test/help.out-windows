Smoke: a framework for testing most things from the very edges.

Usage: smoke-exe.exe [--command ARG] ([-c|--color] | [--no-color]) [--bless]
                     TEST-LOCATION...

Available options:
  --command ARG            Specify or override the command to run
  -c,--color               Color output
  --no-color               Do not color output
  --bless                  Bless the results
  -h,--help                Show this help text
