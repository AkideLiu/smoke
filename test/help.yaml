tests:
  - name: help
    args:
      - --help
    stdout:
      - file: io/help.out-unix
      - file: io/help.out-windows
