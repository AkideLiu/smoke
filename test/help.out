Usage: smoke [options] TEST-DIRECTORY COMMAND
    -c, --[no-]color                 Color output
    -h, --help                       Help me
